### DISCUSSION TITLE
Blacklisting words to hide emotes?

### DISCUSSION THREAD

**Discussion by ameanie** [night/betterttv#6822] [Q&A] [answered]
Hello, I have been using bttv to blacklist words to effectively hide the emotes attached to them, but I am wondering if this also completely removes the rest of the message from the chatter as well? I feel like there are many situations where I just cannot see a message from someone that I definitely do not have blocked.

If so, is there a way to just simply block certain emotes/words without blocking the entire message? Or maybe that works and I have another issue going on?

**Comment by night** [comment, marked as answer]
Yes — when a blacklist keyword matches anywhere in a message, the entire message is hidden (via `display: none`). There isn't currently a way to hide just the emote/word while keeping the rest of the message visible. There are other open discussions tracking finer-grained emote control. Closing as answered.
